inc/db_pgsql.php

```diff
@@ -676,13 +681,13 @@ function field_exists($field, $table)
 	/**
 	 * Add a shutdown query.
 	 *
-	 * @param resource The query data.
-	 * @param string An optional name for the query.
+	 * @param resource $query The query data.
+	 * @param string $name An optional name for the query.
 	 */
-	function shutdown_query($query, $name=0)
+	function shutdown_query($query, $name="")
 	{
 		global $shutdown_queries;
-		if($name)
+		if(!empty($name))
```

Comment:
Well, I see no reason to just stay with `if($name)` as I said. It checked what it was supposed to check - if the parameter contains any value.

Also, is this PR finished? I thought it's still WIP as the title says, if I'm not mistaken not all files were changed as you mentioned.